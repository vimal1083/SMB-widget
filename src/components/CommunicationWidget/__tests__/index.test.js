import { render, screen } from '@testing-library/react';
import { ChatButton, Reducer} from '../index'
import renderer from 'react-test-renderer';

test('should render ', () => {
    const tree = renderer
    .create(<ChatButton />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

test('Reducer function should return correct values', () => {
  expect(Reducer({}, { type: "CLICK_OUTSIDE" })).toEqual({ clickedOutside: true})
});