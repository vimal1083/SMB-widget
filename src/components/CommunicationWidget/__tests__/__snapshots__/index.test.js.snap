// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render  1`] = `
<div
  className="circleButton"
>
  <img
    alt=""
    src="chat.png"
  />
</div>
`;
